docs/rfcs/2024-12-06-chat-tags.md

```diff
@@ -0,0 +1,25 @@
+# Chat tags, emoji entry
+
+## Problem
+
+Organize chats.
+
+Stop users from using chat profiles to organize chats.
+
+More fun as we solve groups problem.
+
+## Solution
+
+This is a simple "quality of life" improvement that's long due, as the number of users and conversations grows, that would probably take just a week to ship.
+
+Proposed UX:
+- tags are per profile.
+- preset tags for favourites, contacts, groups and business chats.
+- user defined tags *must* have text and emoji.
+- if some conversation tag (preset or user defined) is not present in the conversation, the tag will not show in the bar, but would still allow selecting it when tagging.
+- there should be some way to delete tag completely, maybe on long-press on tab bar or maybe in the list of tags?
```

Comment:
list of tags is to be somewhere in settings?

Reply:
can preset tags be deleted? probably not

Reply:
The list of tags will show when attaching tags to conversations. The tag probably should be automatically deleted once not used.

Preset tags they are not tags really, they are buttons for existing criteria. Not sure what you mean.